src/java.desktop/windows/native/libawt/windows/awt_Window.cpp

```diff
@@ -3137,7 +3156,7 @@ void AwtWindow::_ModalDisable(void *param)
         AwtWindow::SetAndActivateModalBlocker(windowHWnd, blockerHWnd);
     }
 
-ret:
+
```

Comment:
I recommend deleting this line, since it does not make sense to have two consecutive blank lines here.